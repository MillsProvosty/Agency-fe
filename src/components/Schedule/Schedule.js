import React from "react";
import "./Schedule.scss";
import Nav from "../Nav/Nav";

const Schedule = () => {
    const role = 'volunteer'
    return (
        <section>
            <Nav />
            
        </section>
    )
}

export default Schedule;