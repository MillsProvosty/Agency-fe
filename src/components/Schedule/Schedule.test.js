import React from 'react';
import ReactDOM from 'react-dom';
import Schedule from './Schedule';

it('renders without crashing', () => {
  expect(true).toEqual(true)
});