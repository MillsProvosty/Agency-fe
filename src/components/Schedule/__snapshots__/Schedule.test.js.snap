// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schedule should match snapshot 1`] = `
<styled.section
  className="Schedule"
>
  <Nav />
  <p>
    I am loading
  </p>
</styled.section>
`;

exports[`Schedule should match snapshot when a user is a volunteer 1`] = `
<styled.section
  className="Schedule"
>
  <Nav />
  <p>
    I am loading
  </p>
</styled.section>
`;
